### PR TITLE
docs(readme): improve readme

### PR DESCRIPTION
**Pull request by Favna**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,3 +59,6 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
 <!-- ALL-CONTRIBUTORS-LIST:END -->
 
 This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
+
+[sapphire framework]: https://github.com/sapphire-project/framework
```
Comment (Rexogamer) [inline README.md:63]: Capitalise it
```suggestion
[Sapphire Framework]: https://github.com/sapphire-project/framework
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
-<div style="text-align:center;"><h1>@sapphire/pieces</h1>
-<h3>Sapphire's piece loader</h3></div>
+<div align="center">
+
+![Sapphire Logo](https://cdn.skyra.pw/gh-assets/sapphire.png)
+
+# @sapphire/pieces
+
+Sapphire's piece loader</h4>
```
Comment (Rexogamer) [inline README.md:7]:
```suggestion
Sapphire's piece loader.</h4>
```
also where's the other h4 tag

File: README.md [hunk 3]
```diff
@@ -8,16 +13,21 @@
 [![npm](https://img.shields.io/npm/v/@sapphire/pieces?color=crimson&logo=npm&style=flat-square)](https://www.npmjs.com/package/@sapphire/pieces)
 [![Depfu](https://badges.depfu.com/badges/30f62cdba0c569448b5f93d532dad8e4/count.svg)](https://depfu.com/github/sapphire-project/pieces?project_id=15121)
 
+</div>
+
+## Description
+
+Pieces make up the core of the [Sapphire Framework][]. In the design of [Sapphire Framework][] every command, event, precondition and others in the root "pieces". They are pieces of your code, that make up the whole.
```
Comment (Rexogamer) [inline README.md:20]: If you're trying to suggest commands and such are pieces...
```suggestion
Pieces make up the core of the [Sapphire Framework][]. In the design of [Sapphire Framework][] your commands, events, preconditions and more are "pieces". They are pieces of your code, that make up the whole.
```

**Review by Rexogamer — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Favna) [reply README.md:63]: Prettier is going to lowercase it again

Comment (Rexogamer) [reply README.md:63]: Curse prettier

**Review by kyranet — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -8,16 +13,21 @@
...
 ## API Documentation
 
-For the full API documentation please refer to [the TypeDoc generated documentation](https://sapphire-project.github.io/pieces)
+For the full API documentation please refer to the TypeDoc generated [documentation](https://sapphire-project.github.io/pieces).
 
 ## Buy us some doughnuts
 
-Sapphire Project is open source and always will be, even if we don't get donations. That said, we know there are amazing people who
-may still want to donate just to show their appreciation. Thanks you very much in advance!
+Sapphire Project is and always will be open source, even if we don't get donations. That being said, we know there are amazing people who may still want to donate just to show their appreciation. Thanks you very much in advance!
```
Comment (kyranet) [inline README.md:30]:
```suggestion
Sapphire Project is and always will be open source, even if we don't get donations. That being said, we know there are amazing people who may still want to donate just to show their appreciation. Thank you very much in advance!
```

**Review by kyranet — CHANGES_REQUESTED** [review]

**Review by kyranet — APPROVED** [review]

**Review by Rexogamer — APPROVED** [review]

**Pull request merged**